<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deposit_calc</class>
 <widget class="QDialog" name="deposit_calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>266</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Стандартный калькулятор</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>266</x>
     <y>400</y>
     <width>266</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Кредитный калькулятор</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>532</x>
     <y>400</y>
     <width>268</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Депозитный калькулятор</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма вклада:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Процентная ставка:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cрок размещения (мес):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>0</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Периодичность выплат:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>130</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Пополнения:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_payments">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>7</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Ежедневно</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в месяц</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в квартал</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>В конце срока</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_adds">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>136</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Ежедневно</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в месяц</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в квартал</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в год</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>200</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Снятия:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_cuts">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>206</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Ежедневно</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в месяц</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в квартал</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в год</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Начисленные проценты:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма вклада с процентами:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Удержано налогов:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_term">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_bet">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_deposit_sum">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_sum">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tax_added">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>350</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_bet_added">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_adds">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>140</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_cuts">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>210</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Налоговая ставка:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_tax">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_count">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>280</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string> Рассчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_capital">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>60</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Периодичность капитализации:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_capital">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>68</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Ежедневно</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в месяц</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в квартал</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в год</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
